src/components/Calculator.js

```diff
@@ -15,31 +15,36 @@ const Calculator = () => {
     setObj(newObj);
   };
   return (
-    <div className="calc">
-      <p className="calc-screen">
-        {obj.total}
-        {obj.operation}
-        {obj.next}
-      </p>
-      <button onClick={() => onButtonPress(obj, 'AC')} type="button">AC</button>
-      <button onClick={() => onButtonPress(obj, '+/-')} type="button">+/-</button>
-      <button onClick={() => onButtonPress(obj, '%')} type="button" className="">%</button>
-      <button onClick={() => onButtonPress(obj, '÷')} type="button" className="calc-operation">÷</button>
-      <button onClick={() => onButtonPress(obj, '7')} type="button" className="calc-num">7</button>
-      <button onClick={() => onButtonPress(obj, '8')} type="button" className="calc-num">8</button>
-      <button onClick={() => onButtonPress(obj, '9')} type="button" className="calc-num">9</button>
-      <button onClick={() => onButtonPress(obj, 'x')} type="button" className="calc-operation">x</button>
-      <button onClick={() => onButtonPress(obj, '4')} type="button" className="calc-num">4</button>
-      <button onClick={() => onButtonPress(obj, '5')} type="button" className="calc-num">5</button>
-      <button onClick={() => onButtonPress(obj, '6')} type="button" className="calc-num">6</button>
-      <button onClick={() => onButtonPress(obj, '-')} type="button" className="calc-operation">-</button>
-      <button onClick={() => onButtonPress(obj, '1')} type="button" className="calc-num">1</button>
-      <button onClick={() => onButtonPress(obj, '2')} type="button" className="calc-num">2</button>
-      <button onClick={() => onButtonPress(obj, '3')} type="button" className="calc-num">3</button>
-      <button onClick={() => onButtonPress(obj, '+')} type="button" className="calc-operation">+</button>
-      <button onClick={() => onButtonPress(obj, '0')} type="button" className="calc-num calc-zero">0</button>
-      <button onClick={() => onButtonPress(obj, '.')} type="button">.</button>
-      <button onClick={() => onButtonPress(obj, '=')} type="button" className="calc-operation">=</button>
+    <div className="container">
+      <div className="calculator">
+        <h1>Let&apos;s do some math!</h1>
+        <div className="calc">
+          <p className="calc-screen">
+            {obj.total}
+            {obj.operation}
+            {obj.next}
+          </p>
+          <button onClick={() => onButtonPress(obj, 'AC')} type="button">AC</button>
+          <button onClick={() => onButtonPress(obj, '+/-')} type="button">+/-</button>
+          <button onClick={() => onButtonPress(obj, '%')} type="button" className="">%</button>
```

Comment:
[OPTIONAL] There is a bug when a user tries to use the modulus with 0, as it is a division and the operation is forbidden. I think this should work just as the division by 0, but you would need to modify `operate.js` that was given to you. Just know that you just have to replicate the catch method used in the division by 0 if you want to fix this. :sweat_smile:

![image](https://user-images.githubusercontent.com/79114233/144684627-c3ef7fa8-1317-488d-9c17-0fecb09c08d9.png)